doc/doc-support/lib-function-locations.nix

```
@@ -1,4 +1,4 @@
-{ pkgs ? (import ./.. { }), nixpkgs ? { }}:
+{ pkgs ? (import ./.. { }), nixpkgs ? { }, libsets }:
```

```suggestion
{ pkgs, nixpkgs, libsets }:
```

I don't think this file will be auto-called, making the default both duplicated and dead code.